bobtemplates/hooks.py

```diff
@@ -49,3 +51,22 @@ def get_plone_classifier_version(configurator, question, answer):
     configurator.variables['package.classifier_version'] = '.'.join(an_split[:2])
 
     return answer
+
+
+def get_deploy_scripts(configurator):
+    deploy_scripts = {
+        'pull': requests.get('https://raw.githubusercontent.com/4teamwork/plone-git-deployment/master/deploy/pull'),
+        'after_push': requests.get('https://raw.githubusercontent.com/4teamwork/plone-git-deployment/master/deploy/after_push'),
+        'update_plone': requests.get('https://raw.githubusercontent.com/4teamwork/plone-git-deployment/master/deploy/update_plone')
+    }
+
+    path = os.path.join(os.getcwd(),
+                        'generated',
+                        configurator.variables['package.fullname'],
+                        'deploy')
+
+    os.makedirs(path)
+
+    for k, v in deploy_scripts.iteritems():
```

Comment:
Please use better names for `k` and `v`:
- `k` and `v` or `key` and `value` are not enough descriptive. You could use `filename`, `content` for instance.
- Using short variables with 1 or 2 characters is not good in general. Apart from beeing ambiguous it is hard to debug in pdb because often the names conflict with the pdb commands.